ios/FluentUI/Navigation/Shy Header/ShyHeaderController.swift

```diff
@@ -475,8 +475,8 @@ class ShyHeaderController: UIViewController {
     ///
     /// - Parameter progress: progress, represented as a fraction (0.5) not a percent (50.0)
     private func updateHeader(with progress: CGFloat, expanding: Bool) {
-        //fall out to ignore repeated animations resulting in the same layout
-        guard shyHeaderView.exposure.progress != progress else {
+        // fall out to ignore repeated animations resulting in the same layout
+        guard shyHeaderView.exposure.progress != progress, msfNavigationController?.allowResizeOfNavigationBarOnScroll == true else {
```

Comment:
nit, it's a boolean, do we need the `== true`?

Reply:
it is needed because msfNavigationController is optional